cdap-ui/app/features/login/login.html

```diff
@@ -56,8 +56,11 @@
   <div class="form-group">
     <div class="col-sm-offset-2 col-sm-10">
       <button type="submit" class="btn btn-primary" ng-disabled="submitting || loginForm.$invalid">
-        Submit
+        Login
       </button>
+      <span class="text-danger">
+        {{ error }}
```

Comment:
nit: Can we use `ng-bind`. In the worstcase something breaks or network is super slow we don't want to end up seeing `{{error}}` in UI indicating. ng-bind prevents (this is from the instances bug we had in workers view).